.github/workflows/release.yml

```
@@ -0,0 +1,31 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: "Tag to be released (e.g., v1.0.0)"
+        required: true
+jobs:
+  prod-container-build:
+    permissions:
+      contents: read
+      packages: write
+    uses: scality/workflows/.github/workflows/docker-build.yaml@v2
+    with:
+      name: cosi
+      namespace: ${{ github.repository_owner }}
+      tag: ${{ github.event.inputs.tag }}
```

`github.event` is an old way to retrieve the inputs, use the following instead:
```suggestion
      tag: ${{ inputs.tag }}
```